zombienet/0001-parachains-smoke-test/0001-parachains-smoke-test.toml

```diff
@@ -2,27 +2,41 @@
 timeout = 1000
 
 [relaychain]
-chain = "rococo-local"
+chain = "wococo-local"
 default_command = "polkadot"
 
   [[relaychain.nodes]]
   name = "alice"
-  args = [ "-lruntime=debug,parachain=trace" ]
 
   [[relaychain.nodes]]
   name = "bob"
-  args = [ "-lruntime=debug,parachain=trace" ]
-  command = "kagome"
-  prometheus_prefix = "kagome"
-
-[[parachains]]
-id = 100
-add_to_genesis = false
-
-  [parachains.collator]
-  name = "collator01"
-  command = "adder-collator"
-  args = [ "-lruntime=debug,parachain=trace" ]
+
+  [[relaychain.nodes]]
+  name = "charlie"
+
+  [[relaychain.nodes]]
+  name = "dave"
+
+  [[relaychain.nodes]]
+  name = "eve"
+
+  [[relaychain.nodes]]
+  name = "ferdie"
+
+  [[relaychain.nodes]]
+  name = "one"
+
+  [[relaychain.nodes]]
+  name = "two"
+
+#[[parachains]]
```

Comment:
Please either add a comment explaining when and how to use the disabled config or remove it.

Reply:
oops, sorry. Committed accidentally